docs/en/install.md

```diff
@@ -104,9 +101,21 @@ These steps will install KubeVela controller and its dependency.
     helm install -n vela-system kubevela kubevela/vela-core --set installCertManager=true
     ```
    
-    If you want to try the latest master branch, try the following command:
+    If you want to try the latest master branch, add flag `--devel` in command `helm search` to choose the latest
+    pre-release versions, like `0.3.2-latest-master`.
```

Comment:
maybe we need to expain more to tell how the `0.3.2-latest-master` generated? And what does it mean?

Reply:
Then we should use version like `v0.3.2-rc` which means the current release is `v0.3.1`, the future one is `v0.3.2`  and this one if the candidate release for `v0.3.2`. With this pattern, I think no doc is needed.